concrete/src/User/UserInfo.php

```diff
@@ -674,7 +707,7 @@ public function hasAvatar()
     }
 
```

Comment:
Following we have some deprecated methods. I added a description to what to use instead those methods (and removed the useless `{@inheritdoc}`: this class does not inherit anything.